plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/Exec.java

```diff
@@ -20,7 +20,7 @@
     private final String[] command;
     private final String   id;
 
-    Exec(String[] command, String id) {
+    public Exec(String[] command, String id) {
```

Comment:
Since OpenShiftConnector is in a different package, we cannot implement `startExec()` or `createExec()` without this class being public -- there seems to be no factory method.